rust/template/distributed_datalog/src/lib.rs

```diff
@@ -5,7 +5,7 @@
     future_incompatible,
     illegal_floating_point_literal_pattern,
     improper_ctypes,
-    intra_doc_link_resolution_failure,
+    broken_intra_doc_links,
```

Comment:
Can you please undo this change too?  It upsets older `rustc`. I have a backwards compatible fix in an upcoming PR.

Other than this the PR looks great.